src/scss/footer.scss

```diff
@@ -39,26 +36,23 @@
   }
 
   &__short-title {
-    display: flex;
```

Comment:
where is flex?

Reply:
he's not needed here

Reply:
![q6](https://user-images.githubusercontent.com/62384781/182589113-0e8df8b5-d3ad-44aa-a890-0736d7120d74.png)
![q5](https://user-images.githubusercontent.com/62384781/182589122-5dc7e54b-17d0-4c17-bf8f-d60abad0d8f1.png)